src/screens/profile/profile.tsx

```diff
@@ -158,7 +164,16 @@ const ProfileScreen: React.FC = () => {
               </ScrollView>
             </View>
             <View style={styles.swiperItem} collapsable={false}>
-              <Text>Education container</Text>
+              <Heading level={HeadingLevel.H5} style={styles.containerHeader}>
+                Education
+              </Heading>
+              <ScrollView showsVerticalScrollIndicator={false}>
+                {educationExperience.map(item => (
+                  <View key={item.id} style={styles.card}>
+                    <EducationCard education={item} />
```

Comment:
You may want to simpliy it a bit by removing additional `View` wrapper and return just `EducationCard`.
To apply layout you want just pass additional `style: StyleProp<ViewStyle>` prop